tools/scripts/pipeline/build/create-tsa-config.ps1

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+<#
+.SYNOPSIS
+Create the tsa.config file based on the parameters. Paramteters will be stored as pipeline variables for confidentiality reasons
+
+.PARAMETER InstanceUrl
+The Url to the TSA instance of the database
+
+.PARAMETER ProjectName
+The name of the TSA project
+
+.PARAMETER CodeBaseAdmins
+The admin[s] of the TSA database, as a semicolon delimited list
+
+.PARAMETER AreaPath
+The Area Path that will be attached to new TSA work items
+
+.PARAMETER IterationPath
+The iteration path that will be attached to new TSA work items
+
+.PARAMETER NotificationAliases
+The email alias[es] that will be used for TSA notifications, as a semicolon delimited list
+
+.PARAMETER Tools
+The tool[s] for which uploads are expected, as a semicolon delimited list
+
+.PARAMETER OutputFile
+The fully qualified path to the output file. Assumes that the folder already exists
+
+.Example Usage 
+.\create-tsa-config.ps1 -InstanceUrl MyInstancUrl -ProjectName MyProject -CodeBaseAdmins Domain/user1;Domain/user2 -AreaPath MyAreaPath -IterationPath -MyIterationPath -NotificationAliases MyTeam@MyCompany.com -Tools CredScan;Roslyn -OutputFolder .\tsa.config
```

Comment:
This didn't quite run as-written. Update with quoting, fix for `-OutputFolder` -> `-OutputFile`, and using [doc format suggested by PowerShell docs](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7.3#comment-based-help-for-a-script):

```suggestion
.EXAMPLE
PS> .\create-tsa-config.ps1 `
        -InstanceUrl "MyInstanceUrl" `
        -ProjectName "MyProject" `
        -CodeBaseAdmins "Domain/user1;Domain/user2" `
        -AreaPath "MyAreaPath" `
        -IterationPath "MyIterationPath" `
        -NotificationAliases "MyTeam@MyCompany.com" `
        -Tools "CredScan;Roslyn" `
        -OutputFile ".\tsa.config"
```